docs/en/observability/profiling-envs.asciidoc

```diff
@@ -0,0 +1,19 @@
+[[profiling-envs]]
+= Environment variables to configure the Universal Profiling Agent
+
+The Universal Profiling Agent can be configured with environment variables.
+
+WARNING: Environment variables do have precedence over command line arguments to the Universal Profiling Agent.
+
+[options="header"]
+|==================================
+| Environment variable | Example | Description
+| `PRODFILER_VERBOSE` | `PRODFILER_VERBOSE=true` | Run the Universal Profiling Agent in verbose mode.
+| `PRODFILER_NO_KERNEL_VERSION_CHECK` | `PRODFILER_NO_KERNEL_VERSION_CHECK=true` | Disable the kernel version check. See <<profiling-no-kernel-version-check, Override kernel version check >> for more details.
+| `PRODFILER_TAGS` | `PRODFILER_TAGS="cloud_region:us-central1;env:staging"` | Set specific tags. See <<profiling-tag-data-query, Tag data for querying>> for more details.
+| `PRODFILER_PROJECT_ID` | `PRODFILER_PROJECT_ID=73` | Set project ID to 73.
+| `PRODFILER_SECRET_TOKEN` | `PRODFILER_SECRET_TOKEN=my_secret_token` | Set the secret token to `my_secret_token`.
+| `PRODFILER_COLLECTION_AGENT` | `PRODFILER_COLLECTION_AGENT=example.com:443` | Set the destination for reporting profiling information to `example.com:443`.
+| `PRODFILER_PROBABILISTIC_THRESHOLD` | `PRODFILER_PROBABILISTIC_THRESHOLD=50` | Set the probabilistic threshold to `50`. See <<profiling-probabilistic-profiling, Probabilistic profiling>> for more details.
+| `PRODFILER_PROBABILISTIC_INTERVAL` |`PRODFILER_PROBABILISTIC_INTERVAL=2m30s` | Set the probabilistic interval to `2m30s`. See <<profiling-probabilistic-profiling, Probabilistic profiling>> for more details.
+|==================================
```

Comment:
We have more configuration options that are also settable via env variables, e.g. `clock-sync-interval`, `map-scale-factor`, `tracers`

Reply:
In the scope of this documentation only environment variables are named and listed, that can be configured on CLI and via the Universal Profiling Agent integration. Listing configuration options that are not documented at this point, should be avoided I think.
Sure, at some point all configuration options should be documented, but this should also be a follow up.